app/src/util/tools.ts

```diff
@@ -597,3 +597,11 @@ export const getInitialCollateral = (networkId: number, collateral: Token): Toke
     }
   }
 }
+
+export const reverseArray = (array: any[]): any[] => {
```

Comment:
A more concise way to do this would be 
const newArray = array.map((e, i, a)=> a[(a.length -1) -i])